pkg/config/jvs_config.go

```diff
@@ -88,3 +95,9 @@ func loadJustificationConfigFromLookuper(ctx context.Context, b []byte, lookuper
 
 	return cfg, nil
 }
+
+// JVSKeyConfig is the config used for KMS Key storing in jvs.
+type JVSKeyConfig struct {
+	// KeyName format: `[projects/*/locations/*/keyRings/*/cryptoKeys/*]`
+	KeyName string `yaml:"key_name,omitempty" json:"key_name,omitempty" firestore:"key_name,omitempty"`
```

Comment:
We don't use json anywhere.

Reply:
We use json in UT. Otherwise viper would fail.

Reply:
Viper supports yaml.

Reply:
I changed it to yaml